src/shared/components/loading-spinner/loading-spinner.module.scss

```diff
@@ -18,16 +18,16 @@
  }
  .loading div {
    position: absolute;
-   width: 36px;
-   height: 36px;
+   width: 30px;
+   height: 30px;
    border-radius: 50%;
-   transform: translate(82px,82px) scale(1);
+   transform: translate(60px,60px) scale(1);
    background: #6f6f6f;
    animation: loading 1.333333333333333s infinite cubic-bezier(0,0.5,0.5,1);
  }
  .loading div:nth-child(1) {
    background: #5c4937;
-   transform: translate(150px,82px) scale(1);
+   transform: translate(111px,60px) scale(1);
```

Comment:
Is `scale(1)` has effect? I think it has no... Correct me if I wrong

Reply:
You right
In this case scale has no effect